kafka-connect-common/src/main/scala/com/datamountaineer/streamreactor/common/config/base/traits/LogOverrideSettings.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017-2023 Lenses.io Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datamountaineer.streamreactor.common.config.base.traits
+
+import cats.implicits.catsSyntaxEitherId
+import ch.qos.logback.classic.Level
+import com.datamountaineer.streamreactor.common.config.base.const.TraitConfigConst._
+import org.apache.kafka.connect.errors.ConnectException
+import org.slf4j.LoggerFactory
+
+import scala.util.Try
+
+trait LogOverrideSettings extends BaseSettings {
+
+  private val CONNECTOR_LOG_LEVEL_OVERRIDE = s"$connectorPrefix.$LOG_LEVEL_OVERRIDE"
+  private val PACKAGES                     = Set("io.lenses", "com.datamountaineer")
```

Comment:
Minor. Scala code would say to use Packages and ConenctorLogLevelOverride (I know that Java takes this approach, but even then these are not static)

Reply:
I would like to address all these style violations en masse.  But probably after some larger pieces of work that are in progress have gone in.  At the moment I will leave it to avoid introducing inconsistencies into the codebase.